scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1216,4 +1220,43 @@
                (api/retrieve-url-path target-framework-id agent-hostname "executor-102")))
         (is (nil? (api/retrieve-url-path target-framework-id agent-hostname "executor-103")))
         (is (= (str "http://" agent-hostname ":5051/files/read.json?path=%2Fpath%2Ffor%2Fexecutor-111")
-               (api/retrieve-url-path target-framework-id agent-hostname "executor-111")))))))
\ No newline at end of file
+               (api/retrieve-url-path target-framework-id agent-hostname "executor-111")))))))
+
+(deftest test-instance-progress
```

Comment:
To me, those two tests seem much less valuable than this one test. For example, if we change how we write to the db, only one of the two proposed tests will fail (the one in `scheduler`), while the other one will continue to pass because it's mocking a now-incorrect write to the database.